polaris.shopify.com/playroom/FrameComponent.tsx

```diff
@@ -10,11 +10,26 @@ export default function FrameComponent({
   theme: any;
   children: React.ReactNode;
 }) {
+  const [features, setFeatures] =
+    useState<React.ComponentProps<typeof AppProvider>['features']>();
   useEffect(() => {
     updateGrowFrameHeight(`${document.body.scrollHeight}px`);
   });
+  useEffect(() => {
+    const urlFeatures = new URLSearchParams(window.location.search).get(
```

Comment:
Yeah, that's correct - if it got rendered server side it would error out (`window is undefined`). Having said that, I'm 90% certain Playroom is fully client side. Doesn't hurt either way.